haas/test_common.py

```diff
@@ -37,13 +37,15 @@ def network_create_simple(network, project):
 
 def newDB():
     """Configures and returns an in-memory DB connection"""
-    init_db(create=True,uri="sqlite:///:memory:")
+    init_db(create=True, uri=cfg.get('database', 'postgres'))
     return Session()
 
 def releaseDB(db):
     """Do we need to do anything here to release resources?"""
-    pass
-
+    db.close_all()
+    # According to the documentation, we shouldn't need the Session().bind, but this
+    # breaks without it.
+    Base.metadata.drop_all(Session().bind)
```

Comment:
Does this work with sqlite as well? (it looks like it should, but please confirm).